init.ps1

```diff
@@ -5,7 +5,7 @@ $SdkPath = $PkgRoot + "/io.improbable.worker.sdk"
 $SdkMobilePath = $PkgRoot + "/io.improbable.worker.sdk.mobile"
 $TestSdkPath="test-project/Packages/io.improbable.worker.sdk.testschema"
 
-$SdkVersion = "14.0.1-b1352-614f42b-WORKER-SNAPSHOT"
+$SdkVersion = "14.0.2"
```

Comment:
We can go back to reading the package.json version here?

Reply:
And in the `sh` script too

Reply:
indeed we can - didn't see the commented out line